device/dell/x86_64-dell_s6000_s1220-r0/plugins/sfputil.py

```diff
@@ -175,9 +197,38 @@ def reset(self, port_num):
         return True
 
     def get_transceiver_change_event(self):
-        """
-        TODO: This function need to be implemented
-        when decide to support monitoring SFP(Xcvrd)
-        on this platform.
-        """
-        raise NotImplementedError
+
+        port_dict = {}
+        port = self.port_start
+
+        # Sleep for a minute
```

Comment:
Sleep for a _second_?

Reply:
Comment "Sleep for a minute" is typo error. I shoud be "Sleep for a second".